changelog.txt

```diff
@@ -21,6 +21,7 @@
 * Update - Bump minimum supported version of WooCommerce from 5.4 to 5.5.
 * Update - Bump minimum required version of WooCommerce from 4.0 to 4.4.
 * Fix - Add credit card on My Account using other payment gateways does not show "Your card number is incomplete" error
+* Fix - Inconsistent shipping options in Payment Request popup
```

Comment:
This is slightly pedantic, but almost all changelog entries have a final `.`, it's nice to keep it consistent, but feel free to ignore.

Reply:
Updated in https://github.com/Automattic/woocommerce-payments/pull/3023/commits/88271b8b5ff3e52e010e7c610e48b1be44c64c6c 🙂